code/game/jobs/job/civilian_vr.dm

```diff
@@ -57,3 +57,39 @@
 /datum/job/chaplain
 	pto_type = PTO_CIVILIAN
 
+
+
+//////////////////////////////////
+//			Entertainer
+//////////////////////////////////
+
+/datum/job/Entertainer
+	title = "Entertainer"
+	flag = ENTERTAINER
+	departments = list(DEPARTMENT_CIVILIAN)
+	department_flag = CIVILIAN
+	faction = "Station"
+	total_positions = 4
+	spawn_positions = 4
```

Comment:
That's more of an IMO, but I feel like that's too many slots for purely RP job?

Reply:
The reasoning is this: 3 people for the play, 1 stage hand. Which really amounts to 1 musician, 2 actors, and a stagehand/narrator. Since it's an rp job, there being that many shouldn't cause a balance issue.

Reply:
I don't plan on changing this.